### ISSUE TITLE
[OneClick] Add visibility rules

### ISSUE THREAD

**Issue by enkol** [mohitbagra/vsts-extensions#32]
It would be nice to be able to set a rule, for when the OnClick-Action should be visible.

Example:
- Bug OneClick-Action "Investigate" has some actions:
  - set State = "Active"
  - set Reason = "Investigate"
  - set AssingedTo = "@me"
  - save WI
- It then also would have a Visibility-Rule:
  - if State in "New,Active" and Reason != "Investigating"

**Comment by mohitbagra** [comment]
You mean only show the action button when the work item form meets certain conditions?

**Comment by enkol** [comment]
@mohitbagra Yes, condition is the better wording. I'd like to see only actions which are contextual relevant, according to the current state of the WI (e.g. don't show "Inestigate" action if the WI is already in investigation state).
